daemon/lokinet-vpn.cpp

```diff
@@ -156,6 +156,10 @@ main(int argc, char* argv[])
     case 0:
       return exit_error(3, "One of --up/--down/--status/--kill must be specified");
     case 1:
+      break;
+    case 2:
+    case 3:
+    case 4:
       return exit_error(3, "Only one of --up/--down/--status/--kill may be specified");
     default:
       break;
```

Comment:
this `return exit_error` should replace the default case here instead of adding 2,3,4 and keeping the existing default case.

Reply:
fixed